modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -1415,6 +1418,7 @@ export abstract class AbstractEthLikeNewCoins extends AbstractEthLikeCoin {
     });
     const transferBuilder = txBuilder.transfer() as TransferBuilder;
     transferBuilder
+      .coin(this.staticsCoin?.name as string)
```

Comment:
Can't we initialise coin as parameter up initialisation / can we figure out that this coin needs to chain prefixes without having to pass in coin name explicitly like this. The reason being, passing in coinName for some and not for others could be confusing for users. We already have _coin which is the coin statics loaded on initilization, so can't we use that where ever we need to take of chainId logics so that it would mean minimal changes one need to make for places which already makes use of transaction builder

Reply:
As discussed setting this coin is actually required for getting the correct chain id and this will be needed only for new coins which will be using chain id so the change is backwards compatible.